LICENSE

```diff
@@ -276,3 +276,9 @@ pekko-http-core contains code based on code by Tim Kienzle with his permission t
 (C) 2000 by Tim Kientzle <kientzle@acm.org>
 
 ---------------
+
+pekko-http-cors contains code that was donated by Lomig Mégard to the Apache Software Foundation
+via a Software Grant Agreement <https://www.apache.org/licenses/contributor-agreements.html#grants>
+See <https://github.com/lomigmegard/pekko-http-cors/issues/33>.
```

Comment:
We don't need this if the code was granted to us

Reply:
if we include something, it should be in the NOTICE file but I don't think we need to include anything

Reply:
Better we have a record somewhere?

Reply:
Since the code is granted to us, I don't see how this differs much a standard PR that we get. And with PRs generally, we do not modify the LICENSE and we insist that all source files are given Apache license headers.

The LICENSE addenda are to point out 3rd party code that has different licenses or that has copyright declarations in those licenses (if the original code was ASL but had a specific copyright in it).

I would favour highlighting the source of the code but I would suggest that we do that in the source files adding something like:

```
// based on code from akka-http-cors <https://github.com/lomigmegard/akka-http-cors> which has been granted
// to the Apache Pekko project - the Software Grant agreement is on record with the Secretary of the ASF
```

We could link to https://github.com/lomigmegard/pekko-http-cors but there is a risk that @lomigmegard could delete that repo.

Reply:
> We could link to https://github.com/lomigmegard/pekko-http-cors but there is a risk that @lomigmegard could delete that repo.

I was thinking of just archiving it, thus any link will still work.